.goreleaser.yml

```diff
@@ -121,19 +94,75 @@ builds:
     goamd64:
       - v1
     prebuilt:
-      path: dist/confluent_{{ .Os }}_{{ .Arch }}_{{ with .Amd64 }}_{{ . }}{{ end }}/confluent
+      path: "prebuilt/confluent_{{ .Os }}_{{ .Arch }}{{ with .Amd64 }}_{{ . }}{{ end }}/confluent"
+  - id: confluent-windows-amd64
+    binary: confluent
+    main: cmd/confluent/main.go
+    flags:
+      - -mod=readonly
+    ldflags:
+      - -s -w -X main.version={{.Env.VERSION}} -X main.commit={{.ShortCommit}} -X main.date={{.Date}} -buildmode=exe  
+    gcflags:
+      - all=-trimpath={{.Env.HOME}}/git
+    asmflags:
+      - all=-trimpath={{.Env.HOME}}/git
+    env:
+      - CGO_ENABLED=1
+      - CC=x86_64-w64-mingw32-gcc
+      - CXX=x86_64-w64-mingw32-g++
+      - CGO_LDFLAGS=${CGO_LDFLAGS} -static
+    goos:
+      - windows
+    goarch:
+      - amd64
+    hooks:
+      pre:
+        - cmd: az login
+        - cmd: az keyvault secret download --file CLIEVCodeSigningCertificate2.pfx --name CLIEVCodeSigningCertificate2 --subscription cc-prod --vault-name CLICodeSigningKeyVault --encoding base64
+        - cmd: xattr -dr com.apple.quarantine ./lib/osslsigncode
+      post:
+        - cmd: ./lib/osslsigncode sign -n "Confluent CLI" -i "https://confluent.io" -pkcs12 CLIEVCodeSigningCertificate2.pfx -in {{ .Path }} -out {{ .Path }}
+        - cmd: rm CLIEVCodeSigningCertificate2.pfx
 
 release:
   disable: "{{.Env.DRY_RUN}}"
 
 archives:
+  - id: binary-alpine
+    format: binary
+    builds:
+      - confluent-alpine-amd64
+      - confluent-alpine-arm64
+    name_template: "{{ .ProjectName }}_{{ .Version }}_alpine_{{ .Arch }}"
+    rlcp: true
+  - id: archive-alpine
+    format: tar.gz
+    builds:
+      - confluent-alpine-amd64
+      - confluent-alpine-arm64
+    name_template: "{{ .ProjectName }}_{{ .Version }}_alpine_{{ .Arch }}"
+    rlcp: true
+    wrap_in_directory: "{{ .ProjectName }}"
+    files:
+      - LICENSE
+      - legal/**/*
   - id: binary
-    name_template: '{{ .ProjectName }}_{{ .Version }}_{{ if eq .Os "linux" }}alpine{{ else }}{{ .Os }}{{ end }}_{{ .Arch }}'
     format: binary
+    builds:
+      - confluent-darwin-amd64
+      - confluent-darwin-arm64
+      - confluent-linux
+      - confluent-windows-amd64
+    name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
     rlcp: true
   - id: archive
-    name_template: '{{ .ProjectName }}_{{ .Version }}_{{ if eq .Os "linux" }}alpine{{ else }}{{ .Os }}{{ end }}_{{ .Arch }}'
     format: tar.gz
+    builds:
+      - confluent-darwin-amd64
+      - confluent-darwin-arm64
+      - confluent-linux
```

Comment:
can't see that in the changes but i assume that it's building all linux binaries

Reply:
Yeah, both of the linux binaries are encompassed by the same ID "confluent-linux"